plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -66,23 +69,47 @@ public String getNodeType() {
     @Property(order = 1, viewable = true)
     @Override
     public String getNodeName() {
-        return classNode.get(nodeName);
+        return this.getNameOrTotal(true);
     }
     
     @NotNull
     @Property(order = 2, viewable = true)
-    public Number getCost() {
-        return cost;
+    public String getIndex(){
```

Comment:
Please fix in other methods as well. You also are missing spaces in control-flow statements (e.g. `if(` -> `if (`)
```suggestion
    public String getIndex() {
```